### DISCUSSION TITLE
Animated sequence diagram

### DISCUSSION THREAD

**Discussion by afrancis-caregility** [mermaid-js/mermaid#4199] [Q&A]
Hi All,
I am trying to customize the sequence diagrams as mentioned in this article.
https://observablehq.com/@tomlarkworthy/animated-sequence-diagrams

Can someone explain to me how this can be done especially with the javascript function. I am unable to get it to work in mermaid live online editor.

**Comment by HappyPaul55** [comment]
This is effect is not done inside mermaid, it is completed outside. So their helper function modifies the source mermaid code before passing it to not Mermaid to render. It is not a feature inside Mermaid nor would I expect it to be. 

To get it working you'll need to run this in your own environment.

**Reply by afrancis-caregility** [reply]
Thanks for the response. I don't want to modify the source code and get it done.
